Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -129,12 +134,25 @@ extension AnimeViewController {
         
     }
     
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
```

Comment:
You don't want to use this. Look into the prefetching API that comes with collectionviews

This is much better for pagination.